locales/de/index.yml

```diff
@@ -135,141 +135,239 @@ global:
+            dontShare: "Nicht freigeben"
+          confirmToast: "Einstellungen aktualisiert!"
+        alert:
+          title: "Du gibst IO nicht die Erlaubnis, deine Nutzungsdaten zu sammeln."
+          body: "Wenn du diese Option bestätigen, kann es sein, dass dem IO-Support-Team bei technischen Problemen nicht alle Daten zur Verfügung stehen, um diese zu lösen."
+        whyBottomSheet:
+          title: "Für welche Zwecke sammeln wir Informationen?"
+          body: "**IO sammelt keine Daten für Marketingzwecke.** \nWir verwenden Tools, die es uns ermöglichen, Daten über deine Nutzung der App aufzuzeichnen, um dir einen besseren Service zu bieten und dich bei Problemen zu unterstützen."
+        securityBottomSheet:
+          title: "Was ist eine Pseudonymisierung?"
+          body: "Sie ist die Verarbeitung personenbezogener Daten in einer Weise, dass die personenbezogener Daten ohne Hinzuziehung zusätzlicher Informationen nicht mehr einer bestimmten betroffenen Person zugeordnet werden können, sofern diese zusätzlichen Informationen gesondert aufbewahrt werden und technischen und organisatorischen Maßnahmen unterliegen, die gewährleisten, dass diese personenbezogener Daten nicht einer identifizierten oder identifizierbaren natürlichen Person zugewiesen werden können."
     appInfo:
-      title: Über IO
-      description: Erfahren Sie mehr über die IO-App
-    developersSectionHeader: Entwickler
+      title: "Über IO"
+      description: "Erfahren mehr über die IO-App"
+    developersSectionHeader: "Entwickler"
     experimentalFeatures:
-      confirmTitle: Experimentelle Merkmale
-      confirmMessage: Experimentelle Funktionen können die Anwendung instabil machen. Sind Sie sicher, dass Sie sie aktivieren möchten?
-      bannerMessage: Sie verwenden experimentelle Funktionen, die möglicherweise unvollständig oder instabil sind.
+      confirmTitle: "Experimentelle Funktionen"
+      confirmMessage: "Experimentelle Funktionen könnten die Anwendung instabil machen. Möchtest du sie wirklich aktivieren?"
+      bannerMessage: "Du verwendst experimentelle Funktionen, die unvollständig oder instabil sein können."
     pagoPaEnvironment:
+      pagoPaEnv: "pagoPA Testumgebung"
+      pagoPAEnvAlert: "Diese Änderung erfordert einen Neustart der App"
+      alertConfirmTitle: "Möchtest du die pagoPA Testumgebung nutzen?"
+      alertConfirmMessage: "Wenn du die pagoPA Testumgebung aktivierst, siehst du deine Transaktionshistorie und deine gespeicherten Zahlungsmethoden nicht mehr und kannst nur noch eigens dafür angelegte Zahlungsmethoden nutzen. Wenn dir nicht klar ist, was dies bedeutet, solltest du diese Funktion nicht aktivieren."
+      alertMessage: "Die App muss neu gestartet werden, damit die Änderungen wirksam werden."
+    pnEnvironment:
+        pnEnv: "PN Testumgebung"
+    appVersion: "App-Version"
+    backendVersion: "Backend-Version"
+    debugMode: "Debug-Modus"
+    designSystem: "Design System"
```

Comment:
it looks like these identifiers have been duplicated, this causes a bug in the in-app locale generation process, may you please remove the duplicated locales? 😄 🙏🏻

Reply:
If you look at https://github.com/dhinterlechner/io-app/blob/profile-german/locales/de/index.yml#L359 it does not seem duplicated. It seems Github is not detecting the change adding line 360-370 while maintaining line 371-379 (old version)

Reply:
i removed
`pnEnvironment:
        pnEnv: "PN Testumgebung"`
which seems to resolve the issue 😊